docs/streams/stream-basics/failover-example.md

```diff
@@ -0,0 +1,357 @@
+---
+sidebar_position: 230
+title: Failover Subscription Example
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Prerequisites from '../../_partials/_prerequisites-sdk-api-key.md';
+
+This page describes how to configure a failover subscription for two or more streams. Refer to the [failover subscription](subscriptions.md#failover) section for details and limitations.
+
+To test the example code, open three terminals simultaneously and run `node producer.js`, then run `consumer-1.js` in second terminal and `consumer-2.js` in third terminal. If successful, you will see messages in both consumer terminals.
+
+The consumers must have the same consumer name.
+
+<Prerequisites />
+
+## Producer Example
```

Comment:
See earlier comment about headings and the consumer bit. Whatever you decide for the first, apply consistently throughout.